astropy/time/tests/test_mask.py

```diff
@@ -16,7 +16,7 @@
                                  atol=2. ** -52 * 24 * 3600)  # 20 ps atol
 is_masked = np.ma.is_masked
 
-# This is r"can't set attribute '{0}'" for Python 3.10, but not 3.9.
+# This is r"can't set attribute '{0}'" for Python 3.10, but not 3.11.
```

Comment:
It would be better to expand the comment to list all three options.

Reply:
Let's not overcomplicate this. pytest will match correctly still with existing logic because even in py310, having the same prefix is enough, any words that come after that is extra. So let's just keep this as-is. I hope this makes sense.